ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -109,6 +109,7 @@
       - [Sflow](#sflow)
       - [SNMP Settings](#snmp-settings)
       - [Monitor Sessions](#monitor-sessions)
+    - [Object Tracking](#object-tracking)
```

Comment:
I think we can move this under ```Monitoring``` as a sub-section. (Any thoughts from second reviewer)

Reply:
Yes please.

Reply:
thanks guys, moving it